doc/source/general/settingsWindows.rst

```diff
@@ -0,0 +1,82 @@
+.. _settingsWindows:
+
+kOS Settings Windows
+====================
+
+.. _applauncher:
+
+kOS Control Panel
+-----------------
+
+As of kOS v0.15, kOS now makes use of Kerbal Space Program's built-in
+Application Launcher toolbar, to open a config/status panel for kOS.
+
+This panel behaves like the other display panels the launcher creates,
+and operates mutually exclusively with them. (For example you can't
+make the kOS App Control Panel appear at the same time as the stock
+Resource display panel. Opening one toggles the other one off,
+and visa versa.)
+
+Here is an annotated image of the control panel and what it does:
+
+.. figure:: /_images/general/controlPanelWindow.png
+    :width: 80 %
+
+Key Notes:
+
+1. Contains every vessel and associated kOS processor that is currently fully loaded into the game.  kOS processors that are not currently in "physics range" because they are far enough away to be unloaded will not appear in the list, as they are not loaded and thus cannot do anything.
+2. kOS version number.
+3. kOS processor part name and name tag.
+4. Power status display.  This is not an interactable button so as not to bypass attempts to lock out control of events/actions.
+5. Toggle button to open or close the terminal window.
+6. Toolbar button, click to toggle the control panel window on and off.
+7. Toggle button to activate or deactivate telnet. See :attr:`Config:TELNET`.
+8. Displays or sets the port that the telnet server will listen on. See :attr:`Config:TPORT`.
+9. Toggle button to enable or disable forcing the telnet to only listen on the local loopback address. See :attr:`Config:LOOPBACK`.
+10. When you hover your cursor over a processor it will be highlighted purple.
+
+.. _settingsWindow:
+
+KSP Difficulty Settings Window
+------------------------------
+
+.. note::
+    .. versionadded:: v1.0.2
+        Previous versions of kOS kept all settings accessible from the App
+        Launcher Window.  KSP version 1.2.0 introduced a new way to
+        store settings within the save file itself, and most settings were
+        migrated to this system/window.
+
+This settings window is accessible when you first start a new game by clicking
+on "Difficulty Options", or in an existing game by clicking on "Difficulty
+Options" from the in game settings menu (accessed by pressing the :kbd:`Escape`
+key, and then clicking "Settings" from the pop up window).
```

Comment:
I want to add a sentence here to make it absolutely clear that these setttings aren't about difficulty at all and that this is just the only place where KSP supports mods adding their own settings in the UI.